TutorLizard.BusinessLogic/Data/DataAccess.cs

```diff
@@ -1,5 +1,21 @@
-﻿namespace TutorLizard.BusinessLogic.Data;
+﻿using Newtonsoft.Json;
+using System.Text.Json;
+
+namespace TutorLizard.BusinessLogic.Data;
 public class DataAccess : IUserIdentityDataAccess, IStudentDataAccess, ITutorDataAccess
 {
     // This class will handle loading and saving data.
+
+    public static void JsonDataAdder<T>(T objectData)
```

Comment:
I agree, it can be static from technical side since it has no references to any fields and properties from dataaccess. But since Dataaccess is some type of service in our architecture and we create instance of this class somewhere top and just inject through constructors, I agree to make it more consistent and remove static decorator from method definition